jetty-util/src/main/java/org/eclipse/jetty/util/IO.java

```diff
@@ -507,6 +507,24 @@ public static long write(GatheringByteChannel out, ByteBuffer[] buffers, int off
         return total;
     }
 
+    /**
+     * Throw an {@link IOException} given a {@link Throwable}, if the supplied throwable is itself an {@link IOException}
+     * it will be thrown, otherwise it will be wrapped with {@link IOException}. If the supplied throwable is a
+     * {@link RuntimeException} or an {@link Error} it will be thrown without being wrapped in {@link IOException}.
+     * @param t the cause of the exception.
+     * @throws IOException if supplied throwable is not RuntimeException or Error.
+     */
+    public static void throwIOException(Throwable t) throws IOException
```

Comment:
You need to work on the name and javadoc a little bit.     It only wraps checked exceptions as `IOException`
So perhaps `throwCheckedIOException` ???